src/hyp3_autorift/process.py

```diff
@@ -348,9 +355,16 @@ def process(reference: str, secondary: str, parameter_file: str = DEFAULT_PARAME
 
         orbits = Path('Orbits').resolve()
         orbits.mkdir(parents=True, exist_ok=True)
-        reference_state_vec, reference_provider = downloadSentinelOrbitFile(reference, directory=str(orbits))
+
+        if (esa_username is None) or (esa_password is None):
+            esa_username, esa_password = get_esa_credentials()
+        reference_state_vec, reference_provider = downloadSentinelOrbitFile(
```

Comment:
```suggestion

        reference_state_vec, reference_provider = downloadSentinelOrbitFile(
```
I think a little whitespace here would improve readability